docs/canvas/canvas-share-workpad.asciidoc

```diff
@@ -2,47 +2,94 @@
+
+[float]
+[[export-single-workpad]]
+=== Export workpads
+
+Create a JSON file of your workpad that you can export outside of {kib}.
+
+. From your workpad, click the *Share workpad* icon in the upper left corner.
+
+. Select *Download as JSON*.
++
+[role="screenshot"]
+image::images/canvas-export-workpad.png[Export single workpad]
+
+Want to export multiple workpads? Go to the *Canvas workpads* page, select the workpads you want to export, then click *Export*.
 
 [float]
 [[create-workpad-pdf]]
 === Create a PDF
 
-To view your workpad outside of Kibana, generate a PDF. 
+Create a PDF copy of your workpad that you can save and share outside of {kib}.
 
 . If you are using a Gold or Platinum license, enable reporting in your `config/kibana.yml` file.
 
 . From your workpad, click the *Share workpad* icon in the upper left corner, then select *PDF reports*.
 
-. Click *Generate PDF*. 
+. Click *Generate PDF*.
 +
 [role="screenshot"]
 image::images/canvas-generate-pdf.gif[Generate PDF]
 
 [float]
-[[export-workpad]]
-=== Export your workpad
+[[create-workpad-URL]]
+=== Create a POST URL
 
-To share your workpad with another author, export it as a JSON file.
+Create the POST URL that automatically generates PDF reports with Watcher or a script.
+
+For more information, refer to <<automating-report-generation, Automating report generation>>.
+
+. If you are using a Gold or Platinum license, enable reporting in your `config/kibana.yml` file.
+
+. From your workpad, click the *Share workpad* icon in the upper left corner, then select *PDF reports*.
+
+. Click *Copy POST URL*.
++
+[role="screenshot"]
+image::images/canvas-create-URL.gif[Create POST URL]
 
 [float]
-[[export-single-workpad]]
-==== Export a single workpad
+[[add-workpad-website]]
+=== Share the workpad on a website
 
-. From your workpad, click the *Share workpad* icon in the upper left corner.
+beta[] To show off your workpad outside of {kib}, share a static version of the workpad on an external website.
 
-. Select *Download as JSON*.
+To make sure that your data remains secure, the data in the JSON file is not connected to {kib}. Canvas does not display elements that manipulate the data on the workpad.
+
+. If you are using a Gold or Platinum license, enable reporting in your `config/kibana.yml` file.
+
+. From your workpad, click the *Share this workpad* icon in the upper left corner, then select *Share on a website*.
+
+. On the *Share on a website* pane, follow the instructions.
 +
 [role="screenshot"]
-image::images/canvas-export-workpad.png[Export single workpad]
+image::images/canvas-embed_workpad.gif[Share the workpad on a website]
+
+NOTE: Shareable workpads encode the current state of the workpad in the JSON file. When you make changes to the workpad, the changes do not appear in the shareable workpad on your website.
 
 [float]
-[[export-multiple-workpads]]
-==== Export multiple workpads
+[[shareable-workpad-enable-autoplay]]
+==== Enable autoplay
```

Comment:
The last two sections aren't about sharing your workpad outside of Kibana.  Do they belong here or might they fit better on a different page.